src/data/balance/starters.ts

```diff
@@ -2,6 +2,12 @@ import { Species } from "#enums/species";
 
 export const POKERUS_STARTER_COUNT = 5;
 
+// #region Friendship constants
```

Comment:
```suggestion
// Friendship constants
```
NIT: Correct me if I'm wrong, but I don't think we use `region` folding anywhere in the code base.

Reply:
There's a couple, but it's not widely used.